engine/apps/alerts/migrations/0010_channelfilter_filtering_term_type.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.17 on 2023-03-07 07:27
+
+from django.db import migrations, models
+from django_add_default_value import AddDefaultValue
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('alerts', '0009_alertreceivechannel_web_templates_modified_at'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='channelfilter',
+            name='filtering_term_type',
+            field=models.IntegerField(choices=[(0, 'regex'), (1, 'jinja2')], default=0),
+        ),
+        # Adding same default value on the database level
+        # for database backwards-compatibility with older versions of code
+        AddDefaultValue(
```

Comment:
this is possible with RunSQL to insert default values, but we'll need to do that manually for each database. django_add_default_value does that for us

Reply:
I didn't get that idea.  Why we need AddDefaultValue, when we have default=0?

Reply:
I left better comment
for more details I recommend to read this [article](../engine/apps/alerts/migrations/0010_channelfilter_filtering_term_type.py)